docs/architecture/adr-075-rpc-subscription.md

```diff
@@ -347,8 +348,8 @@ limit.
 
 The `wait_time` parameter is used to effect polling. If `before` is empty and
 no items are available, the server will wait for up to `wait_time` for matching
-items to arrive at the head of the log. If `wait_time` is zero, the server will
-return whatever eligible items are available immediately.
+items to arrive at the head of the log. If `wait_time` is zero or negative, the
+server will wait for a default (positive) interval.
```

Comment:
Just so I understand this correctly. If for example, my connection dropped and I wanted to catch up on events. I'd send a query with `before` being empty and `after` as the last cursor I'd saved. Say the max results per call was 10 but I was more than 10 behind, wouldn't I want the `wait_time` to be zero so it would instantly give me back all of the 10 events that matched my filter instead of having to wait at least 1 second.